tests/test_utils.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+"""Utils tests."""
+
+from __future__ import absolute_import, print_function
+
+from pytest import raises
+
+from invenio_records_rest.utils import get_default_endpoint_for
+
+
+def test_get_default_endpoint_for():
+    """Test get_default_endpoint_for()."""
+    assert get_default_endpoint_for('recid', {
```

Comment:
Testing `get_default_endpoint_for` was my solution in order to play with different configurations (that I pass as argument), while keeping the code simple.

Otherwise I would have to mock different `RECORDS_REST_ENDPOINTS`, but I don't know how to do it. 

Overall the coverage should be identical.